xdp-ebpf/.cargo/config.toml

```
@@ -0,0 +1,12 @@
+# We have this so that one doesn't need to manually pass
+# --target=bpfel-unknown-none -Z build-std=core when running cargo
+# check/build/doc etc.
+#
+# NB: this file gets loaded only if you run cargo from this directory, it's
+# ignored if you run from the workspace root. See
+# https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure
+[build]
+target = ["bpfeb-unknown-none", "bpfel-unknown-none"]
```

```suggestion
target = ["bpfel-unknown-none"]
```
Loader is hardcoded to load the bpfel version, right?

---

I've removed this file

---

Thank you!